### PR TITLE
Report/de-strip-plots

### PR DESCRIPTION
**Pull request by cmrn-rhi**
- Removed `de_progress_report-1` (can refer to `progress_report` tag `differential-expression-report.md` for it's contents.
- Renamed `de_progress_report-2` documents to `de_progress_report`.
- edited `de_progress_report`:
  - Added top 10 and 20 gene strip plots (tumor vs normal).
  - Remove duplicate `sequence`/`batch_` and `disease_status`/`tumor_status_` columns from report `samples` metadata table.
  - Cleaned up misc. variable names.

Additionally:
- Added to `tumor_normal_DE.R` script to generate strip plots. Unable to test locally, will revert if fails Snakemake check on GitHub.

### CODE REVIEW

**Comment by cmrn-rhi** [comment]
Interested to here form @creisle if this is an easy fix. If not (or at least not obvious) let me know and I'll just revert the `Snakefile` and the R script back to the previous versions.

File: Snakefile [hunk 1]
```diff
@@ -134,9 +134,11 @@ rule de_tumour_normal:
         'results/tumour_de_analysis.p-value_distribution.png',
         'results/tumour_de_analysis.voom.sa-adjusted.png',
         'results/tumour_de_analysis.voom.png',
-        'results/tumour_de_analysis.log2cpm_distribution.png'
+        'results/tumour_de_analysis.log2cpm_distribution.png',
+		'results/tumour_de_analysis.stripplot.tumour_status_t.top_10_genes.png',
```
Comment (creisle) [inline Snakefile:138]: I suspect this indenting is a problem

File: src/tumour_normal_DE.R [hunk 2]
```diff
@@ -242,5 +239,70 @@ pheatmap(
...
+  ### change legend factor names.
+  scale_color_discrete(labels = c("Normal", "Tumor")) + 
+  theme_bw() +
+  ### remove x-axis labels.
+  theme(axis.title.x=element_blank(),
+        axis.text.x=element_blank(),
+        axis.ticks.x=element_blank())
+dev.off()
+
+message('plotting the top 20 ten tumor genes')
```
Comment (creisle) [inline src/tumour_normal_DE.R:288]: This message is confusing? top 20 ten?

**Review by creisle — CHANGES_REQUESTED** [review]
Looks like the snakemake is currently failing

File: Snakefile (hunk 1 above)
Comment (creisle) [reply Snakefile]: I fixed this when I resolved the merge conflicts with master

**Comment by creisle** [comment]
@cmrn-rhi looks like this script has a bug

```generate strip plots
null device 
          1 
Joining, by = "sample_id"
Error: Error in rownames(top10_genes) : object 'top10_genes' not found```

File: src/tumour_normal_DE.R (hunk 2 above)
Comment (cmrn-rhi) [reply src/tumour_normal_DE.R:288]: oh yeah, definitely a typo.

**Comment by FaezeK** [comment]
> @cmrn-rhi looks like this script has a bug
> 
> ```
> null device 
>           1 
> Joining, by = "sample_id"
> Error: Error in rownames(top10_genes) : object 'top10_genes' not found```
> ```

it seems like the name should be top_genes based on line 227

**Comment by cmrn-rhi** [comment]
> > @cmrn-rhi looks like this script has a bug
> > ```
> > null device 
> >           1 
> > Joining, by = "sample_id"
> > Error: Error in rownames(top10_genes) : object 'top10_genes' not found```
> > ```
> 
> it seems like the name should be top_genes based on line 227

Yeah, same with top20_genes. Looks like I didn't clear my work environment when I can it before.

**Review by creisle — APPROVED** [review]

**Pull request merged**